src/main/scala/io/qbeast/spark/delta/writer/BlockWriter.scala

```diff
@@ -20,7 +20,7 @@ import io.qbeast.core.model.IndexFileBuilder
 import io.qbeast.core.model.IndexFileBuilder.BlockBuilder
 import io.qbeast.core.model.TableChanges
 import io.qbeast.core.model.Weight
-import io.qbeast.spark.delta.IndexFiles
+import io.qbeast.spark.delta.QbeastFileUtils
```

Comment:
Why do we need UtilsFiles? Can't we use QbeastFile as an object to provide utility methods to construct?

Reply:
`QbeastFileUtils` contains `delta` imports, that's why I didn't include the logic in the QbeastFile object that is placed in the `core`

Reply:
ok!